### ISSUE TITLE
C按

### ISSUE THREAD

**Issue by 2b-creator** [Captain-Chaos/WorldPainter#272]

**Comment by 2b-creator** [comment]
Can I use Minecraft's custom world to generate terrain via WorldPainter?

**Comment by Captain-Chaos** [comment]
I'm sorry, but I don't understand the question. Can you elaborate? Are you sure this is the right place? The subreddit is better suited for support questions.

**Comment by 2b-creator** [comment]
> I'm sorry, but I don't understand the question. Can you elaborate? Are you sure this is the right place? The subreddit is better suited for support questions.

OK, I see.
That is, Minecraft has a data package to customize the world, its wiki link is at: https://minecraft.fandom.com/wiki/Custom_dimension
It allows players to customize their own dimensions. What I mean is is it possible to generate a world with attributes of the corresponding dimension through Worldpainter (hereinafter referred to as wp)?
Below is an example:
For example, I use wp to create a world, but it only has the original overworld, nether, the_end, (their common point is that the namespace is minecraft), if I want to use the data package to customize the world to generate (such as namespace:test) , Does wp support this function?

**Comment by 2b-creator** [comment]
> I'm sorry, but I don't understand the question. Can you elaborate? Are you sure this is the right place? The subreddit is better suited for support questions.

OK, I see.
That is, Minecraft has a datapack to customize the world, its wiki link is at: https://minecraft.fandom.com/wiki/Custom_dimension
It allows players to customize their own dimensions. What I mean is is it possible to generate a world with attributes of the corresponding dimension through Worldpainter (hereinafter referred to as wp)?
Below is an example:
For example, I use wp to create a world, but it only has the original overworld, nether, the_end, (their common point is that the namespace is minecraft), if I want to use the datapack to customize the world to generate (such as namespace:test) , Does wp support this function?

**Comment by Captain-Chaos** [comment]
No, that is not possible with WorldPainter currently. You can set the world type type to Custom and specify a custom generator name, but that isn't supported for Minecraft 1.18+, and also there is no way to specify the actual generator settings. I'll take this as a feature request, but it would be quite complex to add support for this so don't count on it any time soon.

What may be possible is to generate a map with WorldPainter, and then manually edit the level.dat file with an NBT editor to change the generator settings.
